### PR TITLE
Test infra

### PR DESCRIPTION
**Pull request by MeenuV**
#14 
Working with Travis CI build =)

### CODE REVIEW

**Comment by MeenuV** [comment]
@AliceYuan Yeah - with regards to all the file locations - I wasn't able to figure out imports last night, so I wanted something quick to just test with Travis. I'll clean this up

File: test/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'meenu'
```
Comment (AliceYuan) [inline test/__init__.py]: remove

File: test/client/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__author__ = 'meenu'
```
Comment (AliceYuan) [inline test/client/__init__.py]: remove

File: test/client/app_test.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from client.api import app
+import unittest
+
+
+class FlaskTestCase(unittest.TestCase):
+
+    def test_welcome_page(self):
```
Comment (AliceYuan) [inline test/client/app_test.py]: maybe start page instead? I don't think this page will do anything

**Comment by AliceYuan** [comment]
r+

**Pull request merged**